config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -344,7 +342,7 @@ EOF
         # unrelated to the PR. Testing against the latest Kubernetes is covered
         # by periodic jobs (see https://k8s-testgrid.appspot.com/sig-storage-csi-ci#Summary).
         - name: CSI_PROW_KUBERNETES_VERSION
-          value: "$kubernetes"
+          value: "$kubernetes.0"
```

Comment:
This might be ok for now, but sometimes there are bug fixes or changes to tests made in patch versions that we may want.

Reply:
Yes. Addressing that then will be a bigger change to the `gen-jobs.sh` script again - more that I wanted to tackle now.